src/django_clickhouse/migration_operators.py

```diff
@@ -0,0 +1,48 @@
+from infi import clickhouse_orm
```

Comment:
Nit: No any changes in documentation about migration operations. I don't require it (I can add it myself), though it would be great, if you do.